packages/bpk-component-chip-group/src/Nudger.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type MutableRefObject, useEffect, useState } from 'react';
+
+import { BpkButtonV2, BUTTON_TYPES } from '../../bpk-component-button';
+import { CHIP_TYPES } from '../../bpk-component-chip';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import ArrowLeft from '../../bpk-component-icon/sm/long-arrow-left';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import ArrowRight from '../../bpk-component-icon/sm/long-arrow-right';
+import { cssModules, isRTL } from '../../bpk-react-utils/index';
+import { withButtonAlignment } from '../../bpk-component-icon/index';
+
+import type { ChipStyleType } from './BpkChipGroup';
+import STYLES from './Nudger.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+
+const CHIP_STYLE_TO_BUTTON_STYLE = {
+  [CHIP_TYPES.default]: BUTTON_TYPES.secondary,
+  [CHIP_TYPES.onDark]: BUTTON_TYPES.secondaryOnDark,
+  [CHIP_TYPES.onImage]: BUTTON_TYPES.primaryOnDark,
+}
+
+
+type Props = {
+  chipStyle?: ChipStyleType;
+  scrollContainerRef: MutableRefObject<HTMLElement | null>;
+  leading?: boolean;
+}
+
+
+const AlignedLeftArrowIcon = withButtonAlignment(ArrowLeft);
+const AlignedRightArrowIcon = withButtonAlignment(ArrowRight);
+
+// TODO: how many px to scroll on click?
+const SCROLL_DISTANCE = 100;
```

Comment:
Who would be best to talk to for guidance on this value? Should it be a prop?

Reply:
Design potentially - having a static value might mean that _really_ long chips are not supported as well. 

I could imagine a desire to have a nudge bring the "next" chip into view? https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView maybe?

Reply:
`scrollIntoView` would be really nice, I'm struggling to think of a way to determine which chip to call it on though - you'd need some way to calculate the next chip that is not fully visible in the container. This might be possible but may not be very clean - we can spend a little time exploring it at least.

Reply:
I've dug into this a bit today, it would be possible to support this by using IntersectionObservers on each chip to track which ones are visible then scroll the next one into view, however it does add a lot of complexity to the solution and currently could cause issues if the number of chips change. I've parked it for now but happy to discuss further.

Reply:
After catching up with Adam Wilson, we've decided on a static value of 150px as this feels pretty good and keeps it simple.